js/FlagLoader.js

```diff
@@ -62,7 +80,12 @@ var FlagLoader = {
   },
 
   generateFlags: function FL_generateFlags(flagSuffix) {
-    return {'tracking': 'tracking_' + flagSuffix,
-            'status': 'status_' + flagSuffix};
+    // B2G doesn't use tracking flags, only status
+    if(flagSuffix.indexOf('b2g_') != -1) {
+      return {'status': 'status_' + flagSuffix};
```

Comment:
I'm pretty sure this will cause an exception here:
https://github.com/mozilla/Bugherder/blob/821a27286e4eca667fe26c9844a6a6153257287d/js/bugherder.js#L211

Likely need to sprinkle a conditional or two around to make sure things work even with no tracking status flag extracted.